components/organisms/o-search-panel.vue

```diff
@@ -184,14 +196,16 @@ export default {
     }
     @include for-mobile {
       flex-direction: column;
-      padding: 0 var(--spacer-xl);
+      padding: 0 var(--spacer-sm);
```

Comment:
Looks like this same padding is already set for both desktop and mobile in the container styling. I think we can remove this mobile specific styling all together.

Reply:
I have removed it.